### DISCUSSION TITLE
How to enable closing megamenu by clicking on body

### DISCUSSION THREAD

**Discussion by malle-pietje** [themesberg/flowbite#482] [Q&A]
Hi, new Tailwind user here and loving Flowbite.

I’ve built a dropdown menu based on this variant:
https://flowbite.com/docs/components/mega-menu/#full-width-dropdown

Would like to allow the user to close this dropdown by clicking outside it. Is there a clean way to do this?

**Comment by pekka** [comment]
I got this to work using `data-dropdown-toggle` instead of  `data-collapse-toggle` in the markup for the element that opens/closes the mega menu. The behaviour is there now.

**Reply by distinctperspective** [reply]
Wow u saved my life
